.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Report issues with core functionality to Gutenberg
+    url: https://github.com/WordPress/gutenberg/issues/new?assignees=&labels=Mobile+App+-+i.e.+Android+or+iOS&projects=&template=Bug_report_mobile.md/
```

Comment:
Directly linking to the [Bug report (Mobile)](https://github.com/WordPress/gutenberg/issues/new?assignees=&labels=Mobile+App+-+i.e.+Android+or+iOS&projects=&template=Bug_report_mobile.md) template makes this link very convenient and easy to use. That said, I note that in the `about` field we mention "bug or enhancement request." For the latter, I believe the Gutenberg project generally expects contributors to use the [Feature request](https://github.com/WordPress/gutenberg/issues/new?assignees=&labels=%5BType%5D+Enhancement&projects=&template=Feature_request.md) template. 

Should we consider linking to the top-level [template page](https://github.com/WordPress/gutenberg/issues/new/choose) instead or do you feel this is a non-issue?

Reply:
That reasoning for updating to the top-level page makes a lot of sense to me, and I've updated in https://github.com/wordpress-mobile/gutenberg-mobile/pull/6484/commits/71e14ac499e0589724b415a4c382d251a1d05885. Thank you for that as well as your other thoughtful suggestions, David! 🙇‍♀️